### ISSUE TITLE
Puppet-lint 3.4.0 when called via pdk3 does not seem to respect all disable check configurations

### ISSUE THREAD

**Issue by rnelson0** [puppetlabs/puppet-lint#135]
## Describe the Bug
A file has two reported issues with indents due to a complex heredoc, though they are false positives. I add `PuppetLint.configuration.send('disable_strict_indent')` to my Rakefile and wrap one of the two blocks with `# lint:ignore:strict_indent` and `# lint:endignore`. One issue is still reported, around the heredoc without the lint:ignore.

## Expected Behavior
I expect that disabling the lint check in my Rakefile will take precedence and disable the check on ALL files

## Steps to Reproduce
1. Create a class with a complicated heredoc (more than `@LABEL` and `|LABEL`) inside it, such as:
```
# shortened from full class for example purposes
class profile::windows::patching {
  $patchwindow_value = @("END" /L)
  {
    "monthly_patch_window_start": "${$monthly_patch_window_start}",
    "monthly_patch_window_end": "${$monthly_patch_window_end}"
  }
  | END
}
```
2. Update the Rakefile's PuppetLint configuration to disable the `strict_indent` check:
```
PuppetLint.configuration.send('disable_relative')
PuppetLint.configuration.send('disable_140chars')
PuppetLint.configuration.send('disable_arrow_alignment')
PuppetLint.configuration.send('disable_only_variable_string')
PuppetLint.configuration.send('disable_variable_scope')
PuppetLint.configuration.send('disable_double_quoted_strings')
PuppetLint.configuration.send('disable_parameter_order')
PuppetLint.configuration.send('disable_legacy_facts')
PuppetLint.configuration.send('disable_strict_indent')
```
3. Run `pdk validate` or `pdk bundle exec puppet-lint site/profile/manifests/windows/patching.pp`
4. Look for a warning about the indent starting on the first line of the heredoc contents (following the `@("END" /L)` line)

## Environment
puppet-lint version: 3.4.0
ruby version: 3.2.2-p53
platform: x64-mingw32

## Additional Context
Disabling the check directly (e.g. `pdk bundle exec puppet-lint site/profile/manifests/windows/patching.pp --no-strict_indent-check`) does produce the expected behavior, where there is no warning about indents

**Comment by rnelson0** [comment]
It looks like .sync.yml in the pdk is used to update both *Rakefile* and *.puppet-lint.rc*, and both `pdk validate` and `pdk bundle exec puppet-lint ...` rely on the *.rc* file. Updating that as well remediates the issue. Apologies!
